lib/MusicBrainz/Server/Edit/Relationship/Delete.pm

```diff
@@ -248,6 +250,22 @@ sub accept {
         $self->data->{relationship}{id}
     ) or return;
 
+    $self->c->model('Link')->load($relationship);
+    $self->c->model('LinkType')->load($relationship->link);
+    $self->c->model('LinkType')->load_documentation($relationship->link->type);
+    my @examples = $relationship->link->type->all_examples;
+
+    if (any { $_->{relationship}->id == $relationship->id } @examples) {
+        my $error = localized_note(
+            N_l('This edit would remove a relationship that is set '.
+            'as an example of its relationship type in the documentation. '.
+            'If you still think this should be removed, please '. 
+            'contact us (see details at {contact_url}).'),
+            {contact_us => $CONTACT_URL},
```

Comment:
The placeholder `contact_url` does not match the variable `contact_us`.
While fixing it, I would suggest to directly link from “contact us”.

```suggestion
            'If you still think this should be removed, please '.
            '{contact_url|contact us}.'),
            {contact_url => $CONTACT_URL},
```

Reply:
Agreed. However, this breaks the link because there's a bug where we run `format_editnote` on the resulting URL, so it ends up as `https://musicbrainz.org/edit/%3Ca%20href=%22https://metabrainz.org/contact%22%20rel=%22nofollow%22%3Ehttps://metabrainz.org/contact%3C/a%3E`. I pushed https://github.com/metabrainz/musicbrainz-server/pull/1514/commits/a46113cd9eb2e9ae481badd59c7bbf04723ce4a9 to resolve that.